dreck.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/mholt/caddy/caddyhttp/httpserver"
 	yaml "gopkg.in/yaml.v2"
+	"strings"
```

Comment:
should be with the other std lib imports

Reply:
hum forgot this one. Will add.